### PR TITLE
Update CSS data

### PR DESCRIPTION
**Pull request by yisibl**

### CODE REVIEW

File: features-json/css-variables.json [hunk 1]
```diff
@@ -127,9 +127,9 @@
       "45":"n",
       "46":"n",
       "47":"n",
-      "48":"u",
-      "49":"u",
-      "50":"u"
+      "48":"n",
+      "49":"y #1",
```
Comment (cvrebert) [inline features-json/css-variables.json]: This should be `"n d #1"`

File: features-json/css-variables.json [hunk 2]
```diff
@@ -233,7 +233,7 @@
   },
   "notes":"",
   "notes_by_num":{
-    
+    "1":"Must be enable 'chrome://flags/#enable-experimental-web-platform-features' flag"
```
Comment (cvrebert) [inline features-json/css-variables.json]: ```"Enabled through the \"Experimental Web Platform features\" flag in `chrome://flags`"```

File: features-json/text-emphasis.json [hunk 3]
```diff
@@ -82,7 +86,7 @@
       "42":"n",
       "43":"n",
       "44":"n",
-      "45":"n"
+      "45":"y #2"
```
Comment (cvrebert) [inline features-json/text-emphasis.json]: Should be `"n d #2"`

**Comment by yisibl** [comment]
@cvrebert Sorry, my mistake, I have updated.

File: features-json/css-variables.json (hunk 2 above)
Comment (yisibl) [inline features-json/css-variables.json]: Is this the proper use of it?

**Comment by Fyrd** [comment]
Good updates, thanks @yisibl & @cvrebert for the corrections!

**Comment by mkurz** [comment]
Is there a blog post or anything related which says that CSS variables are enabled through the "Experimental Web Platform features" flag in `chrome://flags`? Where did you get this information from?

**Comment by yisibl** [comment]
@mkurz http://crbug.com/465126 

You can download Chrome Canary and test it([css3test.com](http://yisibl.github.io/css3test/)).

**Comment by mkurz** [comment]
@yisibl Thanks! I am following the Chrome bug but didn't look close enough at the last commit it mentions.

**Pull request merged**